Adding a bit about how to update gatsby-tinacma-git to canary (#209)
tinacms/tinacms-site: content/blog/using-tinacms-on-gatsby-cloud.md

## Configuring Git for Cloud Commits ✔️

<tip>
If you are using the gatsby-tinacms-git plugin, make sure to use version: 0.2.16-canary.0 or later! 
</tip>

> It's simple to setup the canary, just need to run `yarn add gatsby-tinacms-git@canary` until this version reaches a full release.

Reviewer: ```suggestion
To set up for canary, run `yarn add gatsby-tinacms-git@canary` until this version reaches a full release.
```
